### PR TITLE
Sponsorship

### PR DESCRIPTION
**Pull request by ntoll**
**DO NOT MERGE THIS YET**

This branch (stacked upon the tickets branch) introduces the following two changes:
- A new "Sponsorship" page (/sponsorship) based upon old wiki content, Pycon content and Steve's PDF brochure.
- CSS changes for the above page.

THIS PAGE NEEDS TO BE LINKED TO IN SOME WAY! (Ideas in the comments please)

### CODE REVIEW

File: sponsorship/index.html [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+                    <td>£1000</td>
+                    <td>Sponsor banners will be placed in the restaurant and
+                    any marketing material will be placed on the dining
+                    tables.</td>
+                </tr>
+                <tr>
+                    <td>Friday Evening Charity Social</td>
+                    <td>£1500</td>
+                    <td>Sponsor banners will be placed in the venue. All
+                    money raised from delegates will be donated to Cancer UK
```
Comment (inglesp) [inline sponsorship/index.html]: Do we mean "Cancer Research UK"?

File: sponsorship/index.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+                </tr>
+            </table>
+
+            <h2 id="levels">Sponsorship Levels</h2>
+
+            <p>The following levels of sponsorship are available (separately
+            to the named sponsorship items listed above):</p>
+
+            <ul>
+                <li><strong>GOLD</strong>: £10,000</li>
```
Comment (inglesp) [inline sponsorship/index.html]: Can we be consistent about our use of comma separators?

File: sponsorship/index.html [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+            <h2>Contact Us</h2>
+
+            <p>To ask about sponsorship, email us at
+            <a href="mailto:pyconuk-adm@python.org?subject=Sponsorship enquiry">pyconuk-adm@python.org</a>.</p>
+
+            <h2>Sponsor's Deliveries</h2>
+
+            <p>If you're having anything delivered to the conference venue,
+            for example, exhibition materials or swag, then please address it
+            to:</p>
```
Comment (inglesp) [inline sponsorship/index.html]: Say something about ensuring that they're clearly marked for PyCon UK?

**Comment by ntoll** [comment]
OK... all fixed as requested. Please do a final check! ;-)

Thanks!

**Comment by d0ugal** [comment]
LGTM.

**Pull request merged**